addons/costume-editor-shortcuts/addon.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "Costume Editor Shortcuts",
+  "description": "Adds keyboard shortcuts to costume editor tools (similar to TurboWarp). Hover over a tool to view its shortcut.",
+  "credits": [
+    {
+      "name": "Hollow_Song's Dad",
+      "link": "https://scratch.mit.edu/users/Hollow_Song/"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["projects"]
+    }
+  ],
+  "versionAdded": "1.41.0",
+  "tags": ["editor", "costumeEditor"],
+  "enabledByDefault": false
```

```suggestion
  "enabledByDefault": true
```
This one will need some discussion.

---

Hmm... This is a tough one

---

Since it's a new addon, it seems safer to me to not enable it by default until it's proven itself stable, and then maybe enable by default in a future release?